### DISCUSSION TITLE
Where to extract obs-multi-rtmp-X.X.X.X-windows-x64.zip

### DISCUSSION THREAD

**Discussion by schow010** [sorayuki/obs-multi-rtmp#291] [Q&A]
Hi
Need help.
I can't run the exe file but I already installed the previous version. I downloaded the zipped file and it contains 2 subfolder: 1) data 2) obs-lugins.... but I don't know where should I put this? I am using Windows 10 64bit system.
Many Thanks
Stephen

**Comment by sorayuki** [comment]
If you already installed the old version, you should first remove it. You can either use the installer to do the uninstalling, or manually delete the C:\ProgramData\obs-studio\plugins\obs-multi-rtmp folder.
The zip archive has different path structure to the installer. It is designed for extract into obs's install directory (not the one that EXE located).
________________________________
From: schow010 ***@***.***>
Sent: Monday, May 15, 2023 11:52:08 AM
To: sorayuki/obs-multi-rtmp ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [sorayuki/obs-multi-rtmp] Where to extract obs-multi-rtmp-X.X.X.X-windows-x64.zip (Discussion #291)


Hi
Need help.
I can't run the exe file but I already installed the previous version. I downloaded the zipped file and it contains 2 subfolder: 1) data 2) obs-lugins.... but I don't know where should I put this? I am using Windows 10 64bit system.
Many Thanks
Stephen

―

**Reply by schow010** [reply]
Thanks for your help!
